src/actions/install-application/actionLaunch.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import process = require('process');
+import { main } from './index';
+
+process.env['INPUT_USER-NAME'] = process.env['PA_BT_ORG_USER'] ?? 'davidjen@ppdevtools.onmicrosoft.com';
+// process.env['INPUT_APP-ID'] = process.env['PA_BT_ORG_SPN_ID'] ?? '8a7729e0-2b71-4919-a89a-c789d0a9720a';
```

Comment:
Should these commented lines be removed, or just kept as-is while testing the action?

Reply:
it's the latter, yes